core/src/main/java/com/linecorp/armeria/client/DefaultEventLoopScheduler.java

```diff
@@ -290,4 +295,12 @@ public boolean equals(@Nullable Object obj) {
             return ipOrHost.equals(that.ipOrHost) && port == that.port && isHttp1 == that.isHttp1;
         }
     }
+
+    private void lock() {
```

Comment:
nit:
I personally prefer inlining the method if it's used only once in a class. 😄